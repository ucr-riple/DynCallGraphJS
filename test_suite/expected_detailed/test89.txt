{
    'global (test89.js:10:14:10:41)': [
        'Proxy (Native)(test89.js:10:14:10:41)'
    ],
    'global (test89.js:11:1:11:20)': [
        'anon (test89.js:5:15:7:4)'
    ]
}