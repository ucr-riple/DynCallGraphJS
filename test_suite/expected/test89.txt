{ '(test89.js:10:14:10:41)': [ 'Proxy (Native)' ],
  '(test89.js:11:1:11:20)':
   [ '(test89.js:5:15:7:4)' ] }